security/middleware.py

```diff
@@ -137,30 +168,36 @@ def load_setting(self, setting, value):
             return
         value = value.lower()
         if value not in XssProtectMiddleware.OPTIONS.keys():
-            raise ImproperlyConfigured(XssProtectMiddleware.__name__ + " invalid option for XSS_PROTECT.")
+            raise ImproperlyConfigured(
+                XssProtectMiddleware.__name__ +
+                " invalid option for XSS_PROTECT.",
+            )
+
         self.option = value
 
     def process_response(self, request, response):
         """
         Add X-XSS-Protection to the reponse header.
         """
-        response['X-XSS-Protection'] = XssProtectMiddleware.OPTIONS[self.option]
+        response['X-XSS-Protection'] = \
```

Comment:
> The behaviour when you have a space after that trailing slash is a broken file, right?

Yes, although any trailing whitespace is generally bad to have. I have my editor set up to highlight this so that I don't commit it.